website/pages/docs/cheatsheet.mdx

```diff
@@ -56,14 +56,14 @@ Add dependency hosted on a Git repository:
 
 ```toml copy
 [dependencies]
-quaireaux = { git = "https://github.com/keep-starknet-strange/quaireaux.git" }
+quaireaux_math = { git = "https://github.com/keep-starknet-strange/quaireaux.git" }
 ```
 
 Add dependency located in local path:
 
 ```toml copy
 [dependencies]
-quaireaux = { path = "path/to/quaireaux" }
+quaireaux_math = { path = "path/to/quaireaux_math" }
```

Comment:
```suggestion
quaireaux_math = { path = "../path-to-quaireaux-checkout/quaireaux/quaireaux_math" }
```

I wonder if this subtle suggestion to point to subdirectory wouldn't be helpful for people, wdyt?